icechunk-python/python/icechunk/__init__.py

```diff
@@ -134,3 +136,5 @@
     "s3_store",
     "tigris_storage",
 ]
+
+initialize_logs()
```

Comment:
Is this ok? Is this too much? I need to initialize the logging system, but I only want to do that in the "application", not in the Rust library, because users of the library may want to do something different. Since we don't have the entrypoint to the python process, I'm doing this on import of icechunk.

Thoughts?

Reply:
If the default is to print nothing to stdout, it seems like this could be gated on the presence of the `ICECHUNK_LOG` env var.

Reply:
Currently, the default is to ERROR level logging. That's what you get if you don't define the ICECHUNK_LOG variable. The issue with making it easy no to to call this is that hen, error messages don't get the call stack. The mechanism for rich error messages depends on logs being enabled. The can be enabled with no logging, but we have to call the initialize function

Reply:
OK in that case I support enabling logs always. The errors are great now.